### DISCUSSION TITLE
I want to use this package with Next JS but I'm not able.

### DISCUSSION THREAD

**Discussion by dokata99** [joeattardi/picmo#220] [Ideas]
I'm not able to use this package because the package is using localstorage. In NextJS there is no localstorage, because is SSR. Is there a way to fix this or ignore this error. 

![image](https://user-images.githubusercontent.com/72043010/181462955-aecf74d9-5294-4877-9e82-996a96bf3b82.png)

**Comment by joeattardi** [comment]
PicMo does not support server side rendering at this time. To use with Next.js you will have to make sure the picker code only runs in the browser:

```
if (process.browser) {
  // create picker here
}
```
